src/docfx/template/TemplateEngine.cs

```diff
@@ -54,29 +57,25 @@ public TemplateSchema GetSchema(SourceInfo<string> schemaName)
                : throw Errors.SchemaNotFound(schemaName).ToException();
         }
 
-        public string RunLiquid(string content, Document file, JObject rawMetadata)
+        public string RunLiquid(Document file, TemplateModel model)
         {
-            // TODO: only works for conceptual
-            var metadata = CreateMetadata(rawMetadata);
-
-            var layout = rawMetadata.Value<string>("layout") ?? "";
+            var layout = model.RawMetadata.Value<string>("layout") ?? "";
             var themeRelativePath = PathUtility.GetRelativePathToFile(file.SitePath, "_themes");
 
             var liquidModel = new JObject
             {
-                ["content"] = content,
-                ["page"] = rawMetadata,
-                ["metadata"] = metadata,
+                ["content"] = model.Content,
+                ["page"] = model.RawMetadata,
+                ["metadata"] = new JObject(),
```

Comment:
why it's empty JObject?

Reply:
because it is not needed, @yufeih can we just remove `["metadata"]`?

Reply:
@live1206 , checked it twice, it should be:
```suggestion
                ["metadata"] = model.PageMetadata,
```